runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightOverlay.java

```diff
@@ -72,6 +73,13 @@ public Dimension render(Graphics2D graphics)
 		MenuEntry menuEntry = menuEntries[last];
 		String target = menuEntry.getTarget();
 		String option = menuEntry.getOption();
+		int type = menuEntry.getType();
+
+		if (type == MenuAction.RUNELITE_OVERLAY.getId() || type == MenuAction.EXAMINE_ITEM_BANK_EQ.getId())
```

Comment:
Can this just be a call to `shouldLeftClickOpenMenu`?

Reply:
We could, but that would make it never show the tooltip if you have single mouse button on. I'm not entirely sure what the behavior should be in that case.